src/ui/components/Library/Team/View/TestRuns/TestRunDetailsPanel/RunStats.module.css

```diff
@@ -0,0 +1,27 @@
+.Pill {
+  display: flex;
+  height: 1.35rem;
+  min-width: 1.35rem;
+  align-items: center;
+  justify-content: center;
+  border-radius: 0.25rem;
+  font-size: 0.75rem;
+  font-weight: 700;
+  padding: 0 0.5rem;
+}
+
+.flakyPill {
+  color: var(--theme-base-100);
+  background-color: var(--testsuites-flaky-color);
+}
+
+.failedPill {
+  color: #fff; /* we want white in both themes, or else the contrast is bad */
```

Comment:
This should still be a style variable.

Reply:
@jonbell-lot23 - do we have an existing variable for this?

Reply:
@ryanjduffy color: var(--theme-base-100);  should work, and it's what we use in the other .failedPill locations